app/src/main/java/com/mailerdaemon/app/placement/PlacementActivity.kt

```diff
@@ -70,9 +69,10 @@ class PlacementActivity : AppCompatActivity(), AccessDatabase {
                     val result = response.body()
                     if (response.isSuccessful && result != null) {
                         shimmer_view_container.visibility = View.GONE
-                        PlacementAdapter.list = result
                         data = result
-                        PlacementAdapter.notifyDataSetChanged()
+                        adapter = PlacementAdapter(data)
```

Comment:
See here no need for another adpter it is already present in fragment, just pass the list.